webapp/client/src/components/ButtonAnchor.js

```diff
@@ -168,7 +179,7 @@ ButtonAnchor.propTypes = {
   url: PropTypes.string,
   name: PropTypes.string,
   download: PropTypes.bool,
-  target: PropTypes.string,
+  target: PropTypes.oneOf(["_self", "_blank"]),
```

Comment:
can we add just a boolean "openInANewWindow"?

Reply:
In fact there are more possibilities than these both, but I will check the suggestions

Reply:
adjust to only use one prop